src/docs.rs

```diff
@@ -88,6 +88,30 @@ pub fn generate_html(
         })
         .collect::<Vec<Link>>()[..];
 
+    let repo_link = match &project_config.repository {
+        Repository::GitHub { repo, .. }
+        | Repository::GitLab { repo, .. }
+        | Repository::BitBucket { repo, .. } => Some(Link {
+            name: "Repository".to_string(),
+            path: repo.clone(),
+        }),
+        Repository::Custom { url } => Some(Link {
+            name: "Repository".to_string(),
+            path: url.clone(),
+        }),
+        Repository::None => None,
+    }
+    .into_iter()
+    .collect::<Vec<Link>>();
+
+    let links = &doc_links
+        .iter()
+        .chain(repo_link.iter())
```

Comment:
I think if we call `into_iter` rather than `iter` we remove the `map` call that clones the link,

Reply:
I try to change line 108 from ` iter()` to` into_iter()` and remove the `map`

I got error `a value of type std::vec::Vec<docs::Link> cannot be built from an iterator over elements of type &docs::Link`

Can I use `cloned()` for remove the `map`? 

```
    let links = &doc_links
        .iter()
        .chain(repo_link.iter())
        .cloned()
        .collect::<Vec<Link>>()
```

Reply:
Sounds good! I was surprised that it didn't work but this isn't performance sensitive code so who cares 👍

Reply:
Thank you @lpil for your recommend.
I try again in this morning  and it work. I can use` into_iter()` when I remove  reference in `doc_links`

This is ok?


<img width="1150" alt="Screen Shot 2564-01-20 at 09 58 28" src="https://user-images.githubusercontent.com/11692854/105121123-26928500-5b06-11eb-9a01-43ffff5bfc71.png">

Reply:
That looks great! I think we can also drop the `collect` on `doc_links` and then remove the `into_iter` when it is referenced on line 106

Reply:
Thank you @lpil 

It has been done :)